core/src/main/scala/org/broadinstitute/dsde/rawls/util/WorkspaceSupport.scala

```diff
@@ -174,6 +173,33 @@ trait WorkspaceSupport {
       case Some(workspace) => op(workspace)
     }
 
+  def getV2WorkspaceContextAndPermissions(workspaceName: WorkspaceName,
```

Comment:
These are copies of the original methods, with the ultimate query (defined in `withV2WorkspaceContext`) modified to the v2 version.